docs/examples/audio/spectrogram.ipynb

```diff
@@ -0,0 +1,272 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Audio spectrogram"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Background\n",
+    "\n",
+    "In this example we will go through the steps to build a DALI audio processing pipeline, including the calculation of a spectrogram. A spectrogram is a representation of a signal (e.g. an audio signal) that shows the evolution of the spectrum of frequencies on the time domain.\n",
+    "\n",
+    "Typically, and in our DALI implementation, a spectrogram is calculated by computing the fast fourier transform (FFT) over a series of overlapping windows extracted from the original signal. The process of dividing the signal in short term sequences of fixed size and applying FFT on those independently is called Short-time Fourier transform (STFT). The spectrogram is then calculated as the power of the complex magnitude of the STFT.\n",
+    "\n",
+    "Extracting short term windows of the original image affects the calculated spectrum by producing aliasing artifacts. This is often call spectral leakage. To control/reduce the spectral leakage effect, we use different window functions when extracting short term windows. Some examples of window functions are: Hann, rectangular, Hanning.\n",
+    "\n",
+    "It is beyond the scope of this example to go deeper into the details of the signal processing concepts we mentioned above."
```

Comment:
Maybe you can provide some reference where the user can learn more.

Reply:
Done